### ISSUE TITLE
Useless `private` access modifier incorrectly detected

### ISSUE THREAD

**Issue by scottjacobsen** [bbatsov/rubocop#2870]
Given this code:

```ruby
class FoosController < ApplicationController
  private

  helper_method \
  def my_helper
    'help me!'
  end
end
```

Robocop warns that the `private` is useless. But `#my_helper` is indeed private.

**Comment by alexdowad** [comment]
Just pushed a fix to my open PR.
